### ISSUE TITLE
1.1.0 uploaded to pypi with no code in it :(

### ISSUE THREAD

**Issue by harlowja** [tomoh1r/ansible-vault#8]
Seems like the newest version uploaded but with no code in it :(

```
$ cd ansible-vault-1.1.0
(.venv) LMDV-JHARLOW:ansible-vault-1.1.0 jxharlow$ tree .
.
├── PKG-INFO
├── README.rst
├── ansible_vault.egg-info
│   ├── PKG-INFO
│   ├── SOURCES.txt
│   ├── dependency_links.txt
│   ├── requires.txt
│   └── top_level.txt
├── setup.cfg
├── setup.py
└── test
    └── test_api.py

2 directories, 10 files
```

**Comment by tomoh1r** [comment]
I check it, sorry.

**Comment by tomoh1r** [comment]
temporary remove 1.1.0 from pypi

**Comment by tomoh1r** [comment]
#9

**Comment by tomoh1r** [comment]
Fix it, and release 1.1.1.

**Comment by tomoh1r** [comment]
No issue? close it.
